Game/Source/GDKTestGyms/Public/NFRConstants.h

```diff
@@ -8,24 +8,44 @@
 
 DECLARE_LOG_CATEGORY_EXTERN(LogNFRConstants, Log, All);
 
+class FMetricDelay
+{
+public:
+
+	FMetricDelay();
```

Comment:
Do we need this constructor?

Reply:
I get errors stating that "no appropriate default constructor available"